homeassistant/components/frontend/www_static/home-assistant-polymer

```diff
@@ -1 +1 @@
-Subproject commit 697f9397de357cec9662626575fc01d6f921ef22
+Subproject commit 2ecd6a818443780dc5d0d981996d165218b2b094
```

Comment:
https://git-scm.com/docs/git-revert

On Wed, Aug 3, 2016 at 7:51 PM, Xorso <notifications@github.com> wrote:

> In homeassistant/components/frontend/www_static/home-assistant-polymer
> <https://github.com/home-assistant/home-assistant/pull/2703#discussion_r73451865>
> :
>
> > @@ -1 +1 @@
> > -Subproject commit 697f9397de357cec9662626575fc01d6f921ef22
> > +Subproject commit 2ecd6a818443780dc5d0d981996d165218b2b094
>
> I am not sure how to remove it from the PR. Any suggestions?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/2703/files/b96cd9b2fc7ff9dc7ac30b1fefc4421af97066a9#r73451865>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC2fZau1C64W7-Zs5GiwgfibDj1qR9k3ks5qcUWlgaJpZM4JaLK6>
> .
>